src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -26,6 +13,18 @@
 using Dynamo.Wpf.ViewModels;
 using DynamoUnits;
 using FontAwesome5;
+using System;
```

Comment:
I would suggest you to check this option in VS as we follow that pattern: 
Options, Text Editor, C#, Advanced, Place System directives first when sorting usings

Reply:
Thanks for that, changed the setting, should be alright from now on